doc/integrator/backend_qgis.rst

```diff
@@ -22,8 +22,8 @@ In the ``cgxp_print`` plugin add the following configuration:
 
 .. code:: javascript
 
-   encodeLayer: { useNativeAngle: true, serverType: 'qgis' }
-   encodeExternalLayer: { useNativeAngle: true, serverType: 'qgis' }
+   encodeLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
+   encodeExternalLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
```

Comment:
In the documentation it's written in lower case, it's not the case?
http://mapfish.github.io/mapfish-print-doc/layers.html#Wms%20Layer